commercetools-models/src/main/java/io/sphere/sdk/products/attributes/AttributeTableRow.java

```diff
@@ -0,0 +1,24 @@
+package io.sphere.sdk.products.attributes;
+
+import javax.annotation.Nullable;
+
+/**
+ * Data for a key value pair of translated attribute label and value.
+ */
+public interface AttributeTableRow {
+    /**
+     * Returns the translated and formatted value of the attribute.
+     *
+     * @return value
+     */
+    @Nullable
+    String getTranslatedValue();
+
+    /**
+     * Returns the translated and formatted value of the label of the attribute.
+     *
+     * @return label
+     */
+    @Nullable
+    String getTranslatedLabel();
```

Comment:
`Translated` is misleading, as even in the comments say it's translated and formatted. Of course `getTranslatedAndFormattedLabel` is both long and ugly. One solution I can think of is to keep just `getLabel` and change the class name to reflect the translated/formatted thing.